libsrc/video/mc6847/graphics/putsprite.asm

```diff
@@ -0,0 +1,168 @@
+;
+; Sprite Rendering Routine
+; original code by Patrick Davidson (TI 85)
+; modified by Stefano Bodrato - Jan 2001
+;
+; PC6001 version
+;
+;
+; $Id: putsprite.asm,v 1.4 2016-07-02 09:01:36 dom Exp $
+;
+
+    SECTION smc_clib
+    PUBLIC  putsprite
+    PUBLIC  _putsprite
+    EXTERN  cpygraph
+    EXTERN  pixeladdress
+
+; __gfx_coords: d,e (vert-horz)
+; sprite: (ix)
+
+
+putsprite:
```

Comment:
Single mode again, perhaps have an override in the target specific directory and delete this one?